### PR TITLE
chore: Update reference documentation for 2022.15.5

### PR DESCRIPTION
**Pull request by ccedric**
Modifications:
- updated simulations page: environment variables
- updated organizations page: payment
- updated reports: print to pdf & shareable links
- updated various screenshots

Not updated:
- tutorials
- payment funnel

### CODE REVIEW

File: content/reference/admin/users/index.md [hunk 1]
```diff
@@ -54,5 +54,6 @@ There are 4 different user roles in Gatling Enterprise:
 | Create Simulation                        |                    |                    | Own team           | Own team           |
 | Administrate Packages                   |                    |                    | Own team           | Own team           |
 | Administrate API Tokens, Users and Teams |                    |                    |                    | Own team           |
+| Subscribe and view Offers |                    |                    |                    | Global System Admin only           |
```
Comment (tpetillot) [inline content/reference/admin/users/index.md:57]: nit: could you align columns?

File: content/reference/user/organization/index.md [hunk 2]
```diff
@@ -77,3 +79,19 @@ Plans view history.
 * **From** - Start date of the plan.
 * **To** - End date of the plan, if there is one.
 * **Credits** - Number of credits awarded each month by the plan.
+
+## Offers
+
+{{< alert warning >}}
+This section is only available to System Admins.
```
Comment (tpetillot) [inline content/reference/user/organization/index.md:86]:
```suggestion
This section is only available to global System Admins.
```

File: content/reference/user/organization/index.md [hunk 3]
```diff
@@ -34,6 +34,8 @@ Click on the pen icon to edit the **Organization name**.
 
 {{< img src="credits-consumption.png" alt="Organization credit informations" >}}
 
+A credit represents a minute of usage of one Gatling load injector.
```
Comment (tpetillot) [inline content/reference/user/organization/index.md:37]: Maybe in info block?

**Review by tpetillot — CHANGES_REQUESTED** [review]

File: content/reference/user/organization/index.md (hunk 2 above)
Comment (slandelle) [reply content/reference/user/organization/index.md:86]: Maybe explain what we mean by "global System Admins" (= System Admin on Team *)?

**Pull request merged**